web/src/subscriptions.js

```diff
@@ -0,0 +1,24 @@
+
+// WindowScrolled Subscription
+const scrolledFx = a => b => [a, b]
+export const WindowScrolled = scrolledFx((dispatch, props) => {
+  const handleScroll = ev => {
+    dispatch([props.action, { ev, scrollY: window.scrollY }])
+  }
+  addEventListener('scroll', handleScroll)
+  return () => {
+    removeEventListener('scroll', handleScroll)
+  }
+})
+
+// PopState Subscription
+const subFx = a => b => [a, b]
```

Comment:
I believe this is the same function as on line 3, you should be able to use it for both subs

Reply:
Nice catch, will fix